scrypto/src/lib.rs

```diff
@@ -69,7 +69,7 @@ pub fn set_up_panic_hook() {
             "<unknown>".to_owned()
         };
 
-        crate::runtime::Logger::error(sbor::rust::format!(
+        crate::runtime::Runtime::panic(sbor::rust::format!(
             "Panicked at '{}', {}",
```

Comment:
Given that we now know it's a panic (by the fact it's in a panic wrapper in the error), what do you think about dispensing with the `Panicked at '{}', {}` stuff and instead go with `"{} ({})"`?

Reply:
I was following Rust's error message format. I'm happy to change it to what you've proposed.